ecosystem/sep-0006.md

```diff
@@ -355,7 +357,9 @@ Example:
 
 #### Transaction Polling
 
-Once the deposit or withdraw request has been made and KYC information has been passed to the anchor via [SEP-12](sep-0012.md), the wallet will need to detect that the anchor has completed the flow by sending funds to the user. The wallet needs to poll the [Transaction History](#transaction-history) endpoint response from the anchor and match the transaction objects returned using the `memo` and `memo_type` parameters used in the deposit or withdraw success response.
+Once the deposit or withdraw request has been made and KYC information has been passed to the anchor via [SEP-12](sep-0012.md), the wallet will need to detect that the anchor has completed the flow by sending funds to the user. 
```

Comment:
>the wallet will need to detect that the anchor has completed the flow by sending funds to the user. 

❓ This says that the wallet will detect by sending funds to the user. Is the intention that the anchor sends funds, and the wallet detects that has occurred? If so I think it should be reworded.
```suggestion
Once the deposit or withdraw request has been made and KYC information has been passed to the anchor via [SEP-12](sep-0012.md), the wallet will need to detect that the anchor has completed the flow. The wallet can detect the anchor has completed the flow by checking if the users account has received the funds. 
```

I'm not sure how this sentence interacts with the paragraph below too, so I might be completely misunderstanding what is happening here.

Reply:
I agree we could reword this, but I think both the current and suggested phrasing only outline the deposit case. I figured the language here was outside of the scope of this PR, but I can reword it here.

Reply:
I've rephrased the language to be more clear. Let me know if you think what I have works